api-java-mixin.raml

```diff
@@ -436,7 +436,7 @@ types:
     (java-extends): 'ProductDataLike, ProductProjectionMixin,
       com.commercetools.api.models.DomainResource<ProductProjection>, com.commercetools.api.models.WithKey'
   ProductVariant:
-    (java-extends): 'com.commercetools.api.models.WithKey'
+    (java-extends): 'com.commercetools.api.models.WithKey, ProductVariantMixin'
   ProductVariantDraft:
```

Comment:
it was missing the getIdentifier method so I created the related classes to make everything works